desc/integrals/_quad_utils.py

```diff
@@ -211,6 +217,28 @@ def uniform(deg):
     return x, w
 
 
+def chebgauss1(deg):
+    """Gauss-Chebyshev quadrature of the first kind with implicit weighting.
```

Comment:
what is meant by implicit weighting?

Reply:
the weight function is assumed to be part of the function to integrate instead of the quadrature weights